libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -174,17 +174,14 @@ private static void processMPEG4FmtpAttribute(
     @Nullable String configInput = fmtpAttributes.get(PARAMETER_MP4V_CONFIG);
     if (configInput != null) {
       byte[] csd = Util.getBytesFromHexString(configInput);
-      ImmutableList<byte[]> initializationData = ImmutableList.of(csd);
-      formatBuilder.setInitializationData(initializationData);
-      Pair<Integer, Integer> dimensions = CodecSpecificDataUtil.parseMpeg4VideoSpecificConfig(csd);
-      formatBuilder.setWidth(dimensions.first);
-      formatBuilder.setHeight(dimensions.second);
+      formatBuilder.setInitializationData(ImmutableList.of(csd));
+      Pair<Integer, Integer> resolution =
+          CodecSpecificDataUtil.getVideoResolutionFromMpeg4VideoConfig(csd);
+      formatBuilder.setWidth(resolution.first);
+      formatBuilder.setHeight(resolution.second);
```

Comment:
Nit: This can be merged with the line above
`formatBuilder.setWidth(resolution.first).setHeight(resolution.second);`